### PR TITLE
Feature/about us hero section

### PR DESCRIPTION
**Pull request by rusiruavb**
1. Get a pull from the upstream's main branch
2. Add colors into variables.

### CODE REVIEW

File: src/modules/AboutUsModules/WhatWeDoSection/WhatWeDoSection.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './WhatWeDoSection.css';
+import items from './WhatWeDoSection.json';
+
+const WhatWeDoSection = () => (
+  <div className="container">
+    <h1 className="text-center mt-4 title">What We Do?</h1>
+    <div className="row mt-4">
+      {items.data.map((item) => (
+        <div className="col-md-3 mb-3" key={item.id}>
```
Comment (gihanrcg) [inline src/modules/AboutUsModules/WhatWeDoSection/WhatWeDoSection.js:10]: extract a new component from this div. use the same file :)

**Comment by gihanrcg** [comment]
Great work @rusiruavb.
Can we refactor the file extention ??? from `.js` to `.jsx` ?

File: src/modules/AboutUsModules/WhatWeDoSection/WhatWeDoSection.js (hunk 1 above)
Comment (rusiruavb) [reply src/modules/AboutUsModules/WhatWeDoSection/WhatWeDoSection.js:10]: Tou mean a sperate component to display the card?

**Comment by rusiruavb** [comment]
> Great work @rusiruavb.
> Can we refactor the file extention ??? from `.js` to `.jsx` ?

Okay, I will change the extention.

File: src/modules/AboutUsModules/WhatWeDoSection/WhatWeDoSection.js (hunk 1 above)
Comment (gihanrcg) [reply src/modules/AboutUsModules/WhatWeDoSection/WhatWeDoSection.js:10]:
```
const card = (item) => {
    <div className="col-md-3 mb-3" key={item.id}>
      ....
    </div>
}
```

Comment (rusiruavb) [reply src/modules/AboutUsModules/WhatWeDoSection/WhatWeDoSection.js:10]: Okay, I will do the change

File: src/modules/AboutUsModules/AboutUsHeroSection/HeroSection.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          <p className="header-content">
+            A student-driven community based on Sri Lanka Institute of Information Technology aiming to bridge the skill
+            gap between an Undergraduate and an Industry Professional.
+          </p>
+        </div>
+      </div>
+      <div className="col-md-6 header-img">
+        <div className="d-flex justify-content-center">
+          <img src={headerImage} alt="About Us header image" className="d-none d-sm-block header-img" />
+        </div>
```
Comment (Shehanka) [inline src/modules/AboutUsModules/AboutUsHeroSection/HeroSection.js:22]: .jsx

File: src/routes/AppRoutes.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  <div>
+    <Router>
+      <Switch>
+        <Route path="/about" component={AboutUs} exact />
+      </Switch>
+    </Router>
+  </div>
+);
+
+export default AppRoutes;
```
Comment (Shehanka) [inline src/routes/AppRoutes.js:15]: .jsx

**Review by Shehanka — CHANGES_REQUESTED** [review]

File: src/modules/AboutUsModules/WhatWeDoSection/WhatWeDoSection.js (hunk 1 above)
Comment (rusiruavb) [reply src/modules/AboutUsModules/WhatWeDoSection/WhatWeDoSection.js]: @gihanrcg I have created seperate component for display cards.

File: src/routes/AppRoutes.js (hunk 3 above)
Comment (rusiruavb) [reply src/routes/AppRoutes.js:15]: Okay, I will changed it

**Comment by rusiruavb** [comment]
1. Change .js files to .jsx

File: src/routes/AppRoutes.js (hunk 3 above)
Comment (rusiruavb) [reply src/routes/AppRoutes.js]: @Shehanka Now I change the js files into jsx.

**Review by Shehanka — APPROVED** [review]

**Pull request merged**